Profiles/mp-PharmaceuticalProduct.xml

```diff
@@ -68,6 +68,7 @@
          <path value="Medication.code"/>
          <short value="MedicationCode"/>
          <definition value="Coding medication in the Netherlands is done on the basis of the G-standaard (issued by Z-index), which is filled under the direction of Royal Dutch Pharmacists Association (KNMP).&#xD;&#xA;&#xD;&#xA;The coded medication can be expressed as:&#xD;&#xA;&#xD;&#xA;* ZI number&#xD;&#xA;* Trade product code (HPK)&#xD;&#xA;* Prescription code (PRK)&#xD;&#xA;* Generic product code (GPK)&#xD;&#xA;&#xD;&#xA;The ZI number represents the trade product and the package size.&#xD;&#xA;The HPK is the trade product as registered and marketed by the manufacturer.&#xD;&#xA;The PRK contains all GPK information supplemented with information needed to prescribe the right product. This level is intended to facilitate generic prescription of drugs.&#xD;&#xA;The GPK defines the pharmaceutical characteristics of a product: ingredients, strengths, units, pharmaceutical form and route of administration.&#xD;&#xA;&#xD;&#xA;Further information about the G-Standaard levels see &lt;https://www.z-index.nl&gt;.&#xD;&#xA;&#xD;&#xA;So-called 90.000.000 numbers are used in local IT systems. The 90.000.000 number is used in accordance with national agreements."/>
+         <comment value="Depending on the context of use, the code that was actually selected by the user (prescriber, dispenser, etc.) will have the coding.userSelected set to true.  As described in the coding datatype: &#34;A coding may be marked as a &#34;userSelected&#34; if a user selected the particular coded value in a user interface (e.g. the user selects an item in a pick-list). If a user selected coding exists, it is the preferred choice for performing translations etc. Other codes can only be literal translations to alternative code systems, or codes at a lower level of granularity (e.g. a generic code for a vendor-specific primary one).&#xD;&#xA;If there is no coding available, it is preferred to omit this coding element. Receiving systems should anticipate on the possibility of receiving a NullFlavor from sending systems."/>
```

Comment:
Add backticks and a period to `.coding.userSelected`, and idem to `.coding` (in '... to omit this coding element.').

Reply:
Replace 'true' by '_true_' (i.e. make it italic).